CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Release Changelog
 
+- [v2.0.1](#v2.0.0)
```

Comment:
Patch releases go under their respective minor release. See v1.8.1 for example.

Reply:
right, thanks